common/Server/eos/core/eos_core.sqf

```diff
@@ -60,8 +60,12 @@ if (!(getmarkercolor _mkr == "colorblack"))then {
 										if (!surfaceiswater _mPos) then {
 											0=[_mPos,units _aGroup,_mkrX,0,[0,20],true,true] call shk_fnc_fillhouse;
 												}else{
-												0 = [_aGroup,_mkr] call EOS_fnc_taskpatrol;
+													if(isClass(configfile >> "lambs_main_Display")) then {
```

Comment:
We should probably do the same format as detecting ace though the mod path config just encase lambs changes